js/config.js

```diff
@@ -6,7 +6,7 @@ let respecConfig = {
   specStatus: "WV",
   specType: "PR",
   pubDomain: "logboek",
-  shortName: "logboek-extensie-template",
+  shortName: "logboek-extensie-object",
   publishDate: "2025-03-31",
   publishVersion: "0.9",
   prevVersion: [],
```

Comment:
```suggestion
  latestVersion: "https://logius-standaarden.github.io/logboek-extensie-object/",
  prevVersion: [],
```
Dit zou de build moeten fixen